### PR TITLE
Project Movies - Laima & Malin

### PR DESCRIPTION
**Pull request by leelalaim**

### CODE REVIEW

File: code/src/components/MovieCard.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  padding-left: 20px;
+  color: white;
+}
+
+.card-image:hover {
+  opacity: 0.3;
+}
```
Comment (jessika-hage) [inline code/src/components/MovieCard.css:17]: Here you could have instead made the hover effect on the whole link (and added a class and hover effect on that one) and then put a rgba background color with opacity in .title-date. Cause now when hovering over the links the effect disappears when you get over the text.

**Review by jessika-hage — APPROVED** [review]
Such a nice project! :D Had just a few tiny comments. Great job!! :D

File: code/src/pages/MovieDetails.jsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  const { id } = useParams()
+  const [pageNotFound, setPageNotFound] = useState(false)
+  const [movies, setMovies] = useState()
+
+  const MOVIE_URL = `https://api.themoviedb.org/3/movie/${id}?api_key=d1212c48c1a2b13b12dd27882d072960&language=en-US`
+  useEffect(() => {
+    fetch(MOVIE_URL)
+      .then((res) => res.json())
+      .then((json) => {
+        console.log(json);
```
[the remark this answers is not in the document]
Comment (KarolCislowski) [reply code/src/pages/MovieDetails.jsx:16]: 🚓 👮🏻

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
Nice project but please remove all console outputs.

**Comment by leelalaim** [comment]
Requested changes were made

**Review by KarolCislowski — APPROVED** [review]
Great work 👍🏻

**Pull request closed without merging**